### DISCUSSION TITLE
My website is all over. it's almost like my html pages doesn't read my css<title>

### DISCUSSION THREAD

**Discussion by ElPuente** [community/community#42450] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi everyone, I am having issues with my website. I uploaded the documents, but for some reason everything is like all over, like I never style it.  can someone please help me?  This is my site.  https://github.com/ElPuente/MySite.  thanks you for helping me. this is my class final.

**Comment by zhou-hao** [comment]
index.html `<link rel="stylesheet" href="stylesheet/style.css">` , the `stylesheet/style.css` not exists？

**Comment by getlost01** [comment]
I have reviewed your code and find that the way of loading CSS is wrong,

Do! in every HTML page replace 
```
<link rel="stylesheet" href="stylesheet/style.css">
``` 
with 
```
<link rel="stylesheet" href="./style.css">
``` 
then your CSS works perfectly.

**Comment by jfullstackdev** [comment]
that's about relative and absolute URLs, I prefer absolute URLs most of the time

**Comment by Harikrushnareddyvangala** [comment]
Use` `<link` rel="stylesheet" href="./style.css">` instead of `<link` rel="stylesheet" href="stylesheet/style.css">` in your html page.

**Comment by ElPuente** [comment]
Thank you.Sent from my iPhoneOn Dec 23, 2022, at 12:51 AM, 老周 ***@***.***> wrote:﻿
index.html <link rel="stylesheet" href="stylesheet/style.css"> , the stylesheet/style.css not exists？

**Comment by ElPuente** [comment]
Thank you Sent from my iPhoneOn Dec 23, 2022, at 3:31 AM, Aagam Jain ***@***.***> wrote:﻿
I have reviewed your code and find that the way of loading CSS is wrong,
Do! in every HTML page replace
<link rel="stylesheet" href="stylesheet/style.css">

with
<link rel="stylesheet" href="./style.css">

then your CSS works perfectly.
